deep_qa/models/multiple_choice_qa/tuple_inference.py

```diff
@@ -175,3 +176,14 @@ def _build_model(self):
         final_output = MaskedSoftmax()(options_probabilities)
 
         return DeepQaModel(input=[question_input, background_input], output=[final_output])
+
+    @overrides
+    def _instance_debug_output(self, instance: TupleInferenceInstance, outputs: Dict[str, numpy.array]) -> str:
+        result = ""
+        result += "Instance: %s\n" % instance.display_string()
+        result += "Label: %s\n" % instance.label
+        # TODO(pradeep): Dealing with only the output layer. Process other layers' outputs as well.
```

Comment:
hey -- I had written one of these yesterday -- I also had this code which is likely too ugly to keep-keep, but if you want to make it better and incorporate it for debugging?
```result += "TupleMatch output: \n"
        tm_output = outputs.get('timedistributed_3', None)
        if tm_output is not None:
            for option in range(self.num_options):
                result += "  option {0}: {1}\n".format(option, str(tm_output[option]))
        result += "\n"
        result += "NoisyOr 1 output: \n"
        noisy_or_1_output = outputs.get('noisyor_1', None)
        if noisy_or_1_output is not None:
            for option in range(self.num_options):
                result += "  option {0}: {1}\n".format(option, str(noisy_or_1_output[option]))
        result += "\n"
        result += "NoisyOr 2 output: \n"
        noisy_or_2_output = outputs.get('noisyor_2', None)
        if noisy_or_2_output is not None:
            for option in range(self.num_options):
                result += "  option {0}: {1}\n".format(option, str(noisy_or_2_output[option]))
        result += "\n"
```